lib/nimble_csv.ex

```diff
@@ -91,6 +91,27 @@ defmodule NimbleCSV do
     defexception [:message]
   end
 
+  @doc """
+  Returns the options used to define this parser/dumper module.
+
+  This function allows you to retrieve the original options and use them
+  as a base for defining new modules with modified options.
+
+  ## Examples
+
+      # Create a new parser based on RFC4180 but with formula escaping
+      NimbleCSV.define(
+        MyApp.CSV,
+        NimbleCSV.RFC4180.options()
+        |> Keyword.merge(
+          escape_formula: %{~w(@ + - = \\t \\r) => "'"},
```

Comment:
For some reason ExDoc needed the `\\t` and `\\r` escaping, otherwise it would produce a broken code snippet by interpreting the tab and carriage return.